.github/config.yml

```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Layer5 and Meshery community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
+  {% if site.github.dark_mode %}
+  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/meshery-celebration-dark.svg)
+  {% else %}
+  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/meshery-celebration-light.svg)
```

Comment:
Update the links with `meshery-kuma`'s path
https://github.com/meshery/meshery-kuma/blob/master/.github/welcome/meshery-celebration-dark.svg
https://github.com/meshery/meshery-kuma/blob/master/.github/welcome/meshery-celebration-light.svg